db/migrate/20201222134529_make_user_id_and_user_type_not_nullable_and_add_index.rb

```diff
@@ -0,0 +1,7 @@
+class MakeUserIdAndUserTypeNotNullableAndAddIndex < ActiveRecord::Migration[6.0]
+  def change
+    change_column_null :authentication_tokens, :user_id, false
+    change_column_null :authentication_tokens, :user_type, false
```

Comment:
Might be worth double checking for `null` values in the production DB before deploying. (There shouldn't be any because there is a constraint at the active record level).